spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -3011,14 +3011,8 @@ private void processSeeks() {
 			while (offset != null) {
 				traceSeek(offset);
 				try {
```

Comment:
Are these based on a different PR? The latest `main` branch line numbers don't match and the code looks different.

Reply:
No, I worked on the remote main branch.

Reply:
But I worked on the same branch as the work [here](https://github.com/spring-projects/spring-kafka/pull/3410), which I've force-pushed now.

Reply:
Ok understood. My bad. I was looking at a different branch. Sorry.